### ISSUE TITLE
Story promos overlap with Feature Phone

### ISSUE THREAD

**Issue by david-boydell** [bbc/simorgh#2063]
**Describe the bug**
The Story Promos overlap when viewed on a Feature Phone using Firefox browser 48. This applies to the Top Story and Story Promo components 

**To Reproduce**

1. Go to /yoruba, /igbo or /pidgin using a feature phone with Firefox 48 (Doro or Nokia 8110)

**Expected behavior**

**Screenshots**
![20190624_095108](https://user-images.githubusercontent.com/36667923/60009980-c4924500-966e-11e9-92af-82b043243285.jpg)
![20190624_095031](https://user-images.githubusercontent.com/36667923/60010088-f3a8b680-966e-11e9-820b-00d6ec5b3387.jpg)

**Smartphone (please complete the following information):**

- Device: DORO7060
- OS: DORO7060
- Browser: Firefox
- Version: 48

**Comment by andrew-nowak** [comment]
Looks like a Firefox Mobile/feature phone issue: Could not replicate on Desktop Firefox 48

![image](https://user-images.githubusercontent.com/10963046/60341912-f75a7700-99a7-11e9-9578-b15c6964d04f.png)

**Comment by andrew-nowak** [comment]
Ongoing investigation - Testing using the Nokia 8110 suggests that while its User Agent reports its browser as a Firefox 48, it does render `@supports (display:grid)` css, which the real Firefox 48 does not.

Evidently this browser has been modified to some degree by KaiOS (the User Agent in the Nokia 8110 does in fact report KAIOS/2.5)

I believe that this browser is attempting to render the grid implementation rather than the fallback, but is failing due to either partial or buggy grid implementation and will attempt to isolate the particular feature(s) that are failing.

**Comment by andrew-nowak** [comment]
This bug seems to also be present on articles; not added by ws fp.

The KaiOS browser (related to but not firefox 48) will execute css in a `@supports (display: grid)` declaration (and every other grid related declaration I have seen). However, _all text, when nested inside a grid, will only exert spacing for its first line._ A promo with a short headline and summary will be displayed correctly, but at such a small width, few promos will qualify.

Minimal reproducable sample:

```html
<p style="display: grid">But surely, SURELY, this browser is capable of handling text wrapping? You can't POSSIBLY be telling me that it just CAN'T SPACE WRAPPED TEXT, just because it's in a grid. That's madness!</p>
<p>This paragraph will have text overlaid/underlaid/be generally illegible</p>
```

This bug is not fixed by wrapping the text inside a non-grid (ie. `display: block` etc.) element.

I have no ideas how to fix this, bar removing all CSS Grid from Simorgh and relying exclusively on the fallback.

**Comment by andrew-nowak** [comment]
One further discovery: the feature phone behaviour above **can** be replicated (and presumably debugged) using Firefox 48.

1) Visit url `about:config` (click through any 'Here be dragons' warnings)
2) Set `layout.css.grid.enabled` to `true`
3) Open a page

**Comment by andrew-nowak** [comment]
Found a working feature detection flag!!

`@supports (grid-template-columns: fit-content(300px) fit-content(300px) 1fr) `

`fit-content` wasn't added (even behind a feature flag) until firefox 51 <https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content>

Would need testing that this is correctly used in other grid-supporting browsers

**Comment by dr3** [comment]
Ok for the feature phone fix 
- Update https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/src/index.jsx
   Where we have `@supports (display: grid) {` replace with some variation of `@supports (grid-template-columns: fit-content(300px) fit-content(300px) 1fr) {`

Testing this locally, i think we can get away with no simorgh changes

**Comment by andrew-nowak** [comment]
Closing as investigation complete, fix ticket is bbc/psammead#751
